models/data.model.js

```diff
@@ -45,3 +45,33 @@ exports.fetchArticles = () => {
       }
     });
 };
+
+exports.fetchComments = (id, order_by) => {
+  let selectCommentsQueryStr = `SELECT * FROM comments`;
+  const queryParams = [];
+
+  if (id) {
+    selectCommentsQueryStr += ` WHERE article_id = $1`;
+    queryParams.push(id);
+  }
+
+  if (order_by) {
+    selectCommentsQueryStr += ` ORDER BY created_at ${order_by}`;
+  } else {
+    selectCommentsQueryStr += ` ORDER BY created_at DESC`;
+  }
```

Comment:
You could also handle this by setting the default in the paramters to desc eg `s.fetchComments = (id, order_by = 'desc') `

Reply:
It would mean you could remove the if block